tests/testutils/utils.go

```diff
@@ -0,0 +1,77 @@
+// This file contains test utils that are general purpose
+// and should one day be moved to a central location for use across all
+// projects.
+package testutils
+
+import (
+	"fmt"
+	"os/exec"
+
+	. "github.com/onsi/gomega"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/projectcalico/felix/fv/containers"
+	"github.com/projectcalico/libcalico-go/lib/api"
+	"github.com/projectcalico/libcalico-go/lib/client"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const vK8sApiserver = "v1.7.3"
+
+func RunK8sApiserver(etcdIp string) *containers.Container {
+	return containers.Run("st-apiserver",
+		fmt.Sprintf("gcr.io/google_containers/hyperkube-amd64:%s", vK8sApiserver),
```

Comment:
One thought (feel free to ignore for now) - would be nice to pass the version as an argument for say, k8s upgrade tests where we want to stop the apiserver and bring it up with a new version.

Reply:
Eh, ignore that.  It's not necessary now and we can do it easily later if needed.

Reply:
Yeah totes agree. Was keeping it a constant for now so that we can figure out how to override it later.